astropy/wcs/wcsapi/low_level_api.py

```diff
@@ -59,10 +59,10 @@ def pixel_to_world_values(self, *pixel_arrays):
         """
         Convert pixel coordinates to world coordinates.
 
-        This method takes `~astropy.wcs.wcsapi.BaseLowLevelWCS.pixel_n_dim` scalars or arrays as
+        This method takes `~astropy.self.wcsapi.BaseLowLevelWCS.pixel_n_dim` scalars or arrays as
```

Comment:
@astrofrog did you mean to find replace all of these?! :wink:

Reply:
Oops no